pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/minion/PinotTaskManager.java

```diff
@@ -220,6 +218,8 @@ private void updateCronTaskScheduler(String tableWithType, Map<String, String> t
         if (!taskToCronExpressionMap.containsKey(existingTaskType)) {
           try {
             _scheduledExecutorService.deleteJob(JobKey.jobKey(tableWithType, existingTaskType));
+            _controllerMetrics.addMeteredTableValue(getCronJobName(tableWithType, existingTaskType),
+                ControllerMeter.CRON_SCHEDULER_JOB_SCHEDULED, -11L);
```

Comment:
What's the meaning of `-11L` here?

Reply:
it's tracking how many scheduled jobs in the controller.

Reply:
I mean why the number `-11L` is used here. Should it be `-1L`?

Reply:
If `-11L` is meaningful, could you put a comment on its description?